nvtabular/workflow/node.py

```diff
@@ -105,6 +105,8 @@ def compute_schemas(self, root_schema):
                 left_operand = operands.pop(0)
 
                 left_operand_schema = _combine_schemas([left_operand])
+                # upstream schema gets lambda applied
+
```

Comment:
Stray comment here from brainstorming solutions to a different problem (I think)